rules/firestore/firestore.rules

```diff
@@ -1,7 +1,7 @@
 service cloud.firestore {
   match /databases/{database}/documents {
     match /{document=**} {
-      allow read, write: if false;
+      allow read, write: if request.auth.uid != null;
```

Comment:
It'd be good to also have a test for a rule where only a specific user is allowed read or write so that another authenticated user can't read or write the secured document.

Reply:
easily done, I'll provide a few additional examples

Reply:
added user_access.test.js with examples of accessing collections under a uid.